### ISSUE TITLE
Select search engine different from default

### ISSUE THREAD

**Issue by pludi** [brave/browser-android-tabs#168]
Please implement the option to use a different search engine (eg. Amazon or Google) from the default (eg. DuckDuckGo or Startpage) for single searches.

**Comment by SergeyZhukovsky** [comment]
@pludi what do you mean? Do you want something like selector of engines when you type something in the url bar?

**Comment by pludi** [comment]
@SergeyZhukovsky Exactly. Basically something like the feature on the desktop version, where when entering a search term in the address bar and hitting enter uses the default search engine, but prefixing it with, for example, `:g` searches Google instead
